lessons/Algebra2/exploring-logarithmic-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+= Exploring Logarithmic Relationships
+
+@description{Students investigate exponential relationships in demographic data about world countries, using an inquiry-based model of sense-making, hypothesizing, modeling and testing.}
+
+@lesson-prereqs{exploring-linear-relationships, lookups}
+
+@keywords{logarithm, base}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Model logarithmic and exponential relationships using functions
+* Use inverse functions to transform logarithmic and exponential functions
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Pyret to model quadratic relationships in data
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* All students should log into @starter-file{program-list} and, open the @online-exercise{https://code.pyret.org/editor#share=1SGzPWywGy63-DaqOVTy39-497lcWaF4B, Countries of the World Starter File}.
+
+| Key Points For The Facilitator
+|
+* TBD
```

Comment:
Might be worth noting that this is an application / extension of logarithms, and that we don't actually offer instruction on logs here?